server/objects/confirm-identity.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable strict */
+
+const { DAO } = require('./dataAccessObject');
+
+module.exports.confirmIdentityDAO = new DAO('moj/juror-record/confirm-identity', {
+  patch: function(body) {
```

Comment:
If a DAO function is empty you don't need to specify it. Creating a DAO actually sets up GET, POST, PATCH, PUT and DELETE with default settings, and you only need to specify a function if you want specific behaviour.

Reply:
(a pass-through function is empty)

Reply:
updated

I did not know that the payloads were injected automatically :-) perfect stuff